collection/Dockerfile

```diff
@@ -61,4 +61,4 @@ EXPOSE 3000
 # Learn more here: https://nextjs.org/telemetry
 # Uncomment the following line in case you want to disable telemetry.
 ENV NEXT_TELEMETRY_DISABLED=1
-CMD ["./entry.sh"]
+CMD ["dumb-init", "./entry.sh"]
```

Comment:
You don't need to change this to `dumb-init` (I tested on my machine and it was fine).

The shebang at the top of the entry.sh file tells the system "please run everything below using dumb-init"

So this becomes like writing "bash ./entry.sh"

Reply:
(I also tested just changing entry.sh and it worked)